### ISSUE TITLE
obsm (embeddings) could be clarified further

### ISSUE THREAD

**Issue by brianraymor** [chanzuckerberg/single-cell-curation#121]
Both @pablo-gar and @jychien report that the [obsm requirements](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/2.0.0/corpora_schema.md#obsm-embeddings) are hard to parse because the requirements are complex and subtle:

For each `str` key, `obsm` stores a `numpy.ndarray` of shape `(n_obs, m)`, where `n_obs` is the number of rows in `X` and `m >= 1`.

To display a dataset in cellxgene Explorer, Curators MUST annotate **one or more** two-dimensional (`m >= 2`) embeddings (e.g. tSNE, UMAP, PCA, spatial coordinates) as `numpy.ndarrays` in `obsm`. The key for the embedding MUST be prefixed with `"X_"`. The text that follows this prefix is presented to users in the *Embedding Choice* selector in cellxgene Explorer.

which simplifies to:

To display a dataset in cellxgene explorer, Curators MUST annotate **one or more** ... embeddings with keys that MUST be prefixed with "X_"`

It might be clearer if _The key for the embedding_ was _The keys for these embeddings_ ?

And implicitly, Curators MAY annotate other embeddings (that the validator will ignore) with keys of their choosing that will be ignored by cellxgene explorer.

**Comment by pablo-gar** [comment]
> And implicitly, Curators MAY annotate other embeddings (that the validator will ignore) with keys of their choosing that will be ignored by cellxgene explorer.

The validator will still check the type of all embeddings (numpy array) to ensure proper conversion to seurat, but will ignore the other requirements if not prefixed with "X_"

**Comment by jychien** [comment]
> It might be clearer if _The key for the embedding_ was _The keys for these embeddings_ ?

Yes, I think that would clarify things.

**Comment by brianraymor** [comment]
Addressed in 63c5649cdead40b9516953e2e260c19dccf1e305
